jobsdb/jobsdb.go

```diff
@@ -2588,6 +2590,10 @@ func (jd *HandleT) GetWaiting(customValFilters []string, count int, parameterFil
 	return jd.GetProcessed([]string{Waiting.State}, customValFilters, count, parameterFilters)
 }
 
+func (jd *HandleT) GetThrottled(customValFilters []string, count int, parameterFilters []ParameterFilterT) []*JobT {
```

Comment:
This seems to be getting called from only one place and this is a one line function. If its not needed in other places, can we just move this functionality to wherever this is being called from?